client/index.html

```diff
@@ -42,6 +42,7 @@
     <script src="./bower_components/angular-favicon/angular-favicon.min.js"></script>
     <script src="./bower_components/jpkleemans-angular-validate/dist/angular-validate.min.js"></script>
     <script src="./bower_components/angular-drag-and-drop-lists/angular-drag-and-drop-lists.min.js"></script>
+	<script src='./bower_components/satellizer/dist/satellizer.js'></script>
```

Comment:
I saw this bag on gitgub with bad formated code line. But in my editor all code lines well formated :)